src/main/java/org/galatea/starter/entrypoint/BaseRestController.java

```diff
@@ -4,21 +4,35 @@
 import net.sf.aspect4log.Log;
 import net.sf.aspect4log.Log.Level;
 import org.galatea.starter.utils.Tracer;
-
+import org.owasp.esapi.ESAPI;
 /**
  * Provides base functionality shared by all Fuse REST controllers
  */
 @Slf4j
 @Log(enterLevel = Level.INFO, exitLevel = Level.INFO)
 public abstract class BaseRestController {
+  protected static final char ENCODER_SEPARATOR = '_';
+
+  /** 
+   * Encodes specified message, to avoid any log forging.
+   */
+  protected String encode(String message){
```

Comment:
Possibly overkill, but we could use the @NonNull annotation on this parameter, given that it should never be called with a null parameter.

Reply:
I would argue that it is overkill, provided that we already check for requestID != null in the first place. However, no harm is added by adding this feature and could serve to enforce the method to never be null.